gravitee-am-ui/src/app/domain/settings/password-policies/password-policies-idp-select-dialog/password-policies-idp-select-dialog.component.html

```diff
@@ -53,6 +53,6 @@ <h2>Add Identity Providers (IdP)</h2>
 
   <div fxLayout="row" fxLayoutAlign="end end" class="action-button-container">
     <button mat-raised-button (click)="closeDialog()">Cancel</button>
-    <button mat-raised-button color="primary" [disabled]="this.result.size === 0" (click)="confirmSelection()">Add Selected</button>
+    <button mat-raised-button color="primary" [disabled]="enabledAdd()" (click)="confirmSelection()">Add Selected</button>
```

Comment:
`disabled=enabled()` feels like an error when reading the code. `[disabled]=!enabledAdd()` or `[disabled]=disabledAdd()` would be more obvious.